sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -1,18 +1,15 @@
-# coding: utf-8
-
 # -------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
+import pytest
```

Comment:
Technically this should go just above `azure.core` since its a library not a built-in